frontend/src/views/Settings/OrgSettingsPage/components/OrgDeleteSection/OrgDeleteSection.tsx

```diff
@@ -1,81 +1,70 @@
 import { useRouter } from "next/router";
 
 import { useNotificationContext } from "@app/components/context/Notifications/NotificationProvider";
-import {
-    Button,
-    DeleteActionModal
-} from "@app/components/v2";
-import { useOrganization, useUser } from "@app/context";
-import {
-    useDeleteOrgById,
-    useGetOrgUsers
-} from "@app/hooks/api";
+import { Button, DeleteActionModal } from "@app/components/v2";
+import { useOrganization, useOrgPermission } from "@app/context";
+import { useDeleteOrgById } from "@app/hooks/api";
 import { usePopUp } from "@app/hooks/usePopUp";
 import { navigateUserToOrg } from "@app/views/Login/Login.utils";
 
 export const OrgDeleteSection = () => {
-    const router = useRouter();
-    const { currentOrg } = useOrganization();
-    const { user } = useUser();
-    const { createNotification } = useNotificationContext();
-    const { data: members } = useGetOrgUsers(currentOrg?._id ?? "");
-    
-    const membershipOrg = members?.find((member) => member.user._id === user._id);
```

Comment:
I understand that this wasn't great and the new approach is better, but it doesn't fully explain the issue we see in prod. What do you think hte issue is? Is it the time it takes to find the member and before it does it errors out on frontend?